docs/hypernode-deploy/pipelines/bitbucket-pipelines.md

```diff
@@ -1,3 +1,101 @@
 # Bitbucket Pipelines
 
-This article is currently work in progress!
+## Configuring deployment environments
+
+To start using Bitbucket Pipelines, we need to prepare the environments we want to deploy to.
+
+For example, these environments can be:
+
+- production
+- acceptance (or staging)
+- test
+
+### Configuring the production environment
+
+Hypernode Deploy will need a pair of SSH keys for authentication to the server.
+
+First, we generate an SSH keypair on the production server, copy the public key to the `~/.ssh/authorized_keys` file
```

Comment:
Bitbucket Pipelines has a default keypair as well right? So perhaps good to clarify this is optional? I haven't verified if that works with Hypernode Deploy, though